### PR TITLE
[add] preliminary support for Arduino MKR1000

### PR DESCRIPTION
**Pull request by agn0stico**
@z3t0, I was almost forgetting about this pull request. There you go.

### CODE REVIEW

File: IRremoteInt.h [hunk 1]
```diff
@@ -87,6 +87,11 @@ EXTERN  volatile irparams_t  irparams;
 #	define BLINKLED_ON()   (PORTD |= B00000001)
 #	define BLINKLED_OFF()  (PORTD &= B11111110)
```
Comment (z3t0) [inline IRremoteInt.h]: This is an example of flipping bits to achieve the same result as digitalWrite but without being as costly

Comment (AnalysIR) [inline IRremoteInt.h]: However, digitalWrite will be more portable between physical Arduino-ish platforms....and will not be as big an issue on faster MCUs.

I suspect it isn't a big issue for 99% of existing users & advanced users can always fine tune for performance where needed.

Comment (Adminius) [inline IRremoteInt.h]: https://github.com/z3t0/Arduino-IRremote/issues/318#issuecomment-285298447

**Comment by bengtmartensson** [comment]
Please close. Not based on current code.

**Pull request closed without merging**